tests/unit/plugins/modules/test_ome_application_certificate.py

```diff
@@ -105,6 +106,26 @@ def test_upload_csr_success(self, mocker, ome_default_args, ome_connection_mock_
         result = self.execute_module(ome_default_args)
         assert result['msg'] == "Successfully uploaded application certificate."
 
+    def test_upload_cert_chain_fail01(self, mocker, ome_default_args, ome_connection_mock_for_application_certificate,
+                                      ome_response_mock):
+        args = {"command": "upload_cert_chain", "upload_file": "/path/certificate_chain.cer"}
+        f_module = self.get_module_mock(params=args)
+        with pytest.raises(Exception) as exc:
+            self.module.get_resource_parameters(f_module)
+        assert exc.value.args[0] == "No such file or directory."
+
+    def test_upload_cert_chain_success(self, mocker, ome_default_args, ome_connection_mock_for_application_certificate,
+                                       ome_response_mock):
+        payload = "--BEGIN-REQUEST--"
+        mocker.patch(MODULE_PATH + 'ome_application_certificate.get_resource_parameters',
+                     return_value=("POST", "ApplicationService/Actions/ApplicationService.UploadCertChain", payload))
+        mocker.patch(MODULE_UTILS_PATH + "RestOME.get_ome_version", return_value="4.1.0")
+        ome_default_args.update({"command": "upload_cert_chain", "upload_file": "/path/certificate_chain.cer"})
+        ome_response_mock.success = True
```

Comment:
In success UT, make sure to assert the success msg, assertion is not right.

Reply:
Done